server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
+  user_id: { type: Number, ref: 'User', required: true },
```

Comment:
For the user_id type, I think we need something like: type: mongoose.Schema.Types.ObjectId, rather than Number.

Reply:
Done!